### DISCUSSION TITLE
When I click "Profile", it automatically searches my email

### DISCUSSION THREAD

**Discussion by doublemultiply** [requarks/wiki#3327] [Error / Bug Report]
**Describe the bug**
I just configured my email settings to enable self-registration.
I made myself a new account on my wiki, and it all works fine, but when I click to navigate to my profile, for some reason it automatically searches the email address I registered with

**To Reproduce**
1. Setup and configure self-registration
2. Register a new user
3. Click on "Profile" at the top right
4. It automatically searches for the email I registered with.

**Expected behavior**
Nothing. I expect a clean profile page without any automatic searches.

**Screenshots**
1. https://i.imgur.com/qb69ym7.png
2. https://i.imgur.com/zgrXHLJ.png

**Host Info (please complete the following information):**
 - OS: Unraid 6.8.3
 - Wiki.js version: 2.5.170
 - Database engine: postgres 12.3

**Additional context**
It's only a minor bug, but still a bug so I thought I'd report it.

**Comment by NGPixel** [comment]
Make sure you don't have any browser extensions that autofill fields

**Reply by LetterN** [reply]
Chrome automatically does the autofill, which is annoying. A way to disable it is to set the `autocomplete ` value to `chrome-off` or `off` (chrome ignores off)

https://stackoverflow.com/questions/15738259/disabling-chrome-autofill

**Comment by doublemultiply** [comment]
> Make sure you don't have any browser extensions that autofill fields

I don't have any extensions that do that, but I'm using Ungoogled Chromium so I assume my browser thinks it's a login field.

**Comment by rajatchauhanyti** [comment]
Hi All, I am also facing the similar issue. the steps to reproduce is straight forward.
After logged in , navigate to profile page, now click on search bar, it will showing up emailId and password as autocomplete suggestion.

I have temporary fix this bug but this issue is weird and i am not able to find the root cause and exact solution.
i have noticed that even "autocomplete" is off but these text-fields are still showing the autocomplete suggestions. 

Temporary fix.
Goto profile page -  client\components\profile\profile.vue 

search for string changepassword, For me it is at 133 line, remove the type='password' from the v-text-field property of changePassword

**Comment by Hagbrain** [comment]
Just turn off autofill in your browsers

**Comment by rajatchauhanyti** [comment]
But we don't want to turn off autofill as we required the autofill feature for all other application as well.

**Reply by NGPixel** [reply]
Most autofill can be disabled on a per-site basis
